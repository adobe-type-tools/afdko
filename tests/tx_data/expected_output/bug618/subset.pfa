%!FontType1-1.1: SourceSerifPro-Regular 1.0
%ADOt1write: (1.0.34)
%%BeginResource: font SourceSerifPro-Regular
12 dict dup begin
/FontType 1 def
/FontName /SourceSerifPro-Regular def
/FontInfo 7 dict dup begin
/version (1.0) def
/Notice (Source is a trademark of Adobe Systems Incorporated in the United States and/or other countries.) def
/Copyright (Copyright 2014 Adobe Systems Incorporated. All Rights Reserved.) def
/FamilyName (Source Serif Pro) def
end def
/PaintType 0 def
/FontMatrix [0.001 0 0 0.001 0 0] def
/Encoding StandardEncoding def
/FontBBox {45 -13 560 669} def
end
currentfile eexec BAB431EA06BB0A1031E1AA11919E714AC69FC6474167B6
17C8FF56014245E5C1290C81D5312994CDC47B07C655DDD53C35D530C44F953F
804152D5ED7591016D3D1968FC4A9F629E0ADB83B91691E8A0AB81C098760580
FFC73B03E534FD00C1C69885C6151AADEBF9BBF7284F088A81EB0A35C6F197CE
C173C8CE58E9B3FBEBF7D49545999FA57A7A5A1B26B5A043EFA3B327BCA94F18
86E0147BE33716D304B24CEB7E89D8626444FA1EE116EEBDBB6E33ADDEEFE14E
2DC12D84FE2170D0C1780958ED3BA07FA26741A6FE7B905C5C7EEBF87FCB87C1
152EDB0A053EABFA5158B67F1A5FCAB305A3629E5742EFE1F03AC8B914674ACF
3561EFE1F3F63FF9DBDD6BB3656F8CAC8FBFED6BD0268D01803B872E3A013E37
5E2A88A23DC6AC16C2DD6BA37DCBFCFB2BE8BC3C0D50AC62B7BDA8EC036FA5B7
B6FDC19EE2F7A8DFDAAAB8848D2767B945658D7A76FC1DC150D2C7C4D439E4C6
5AE8D4C78D4753139A826042642E768DC6BA0A299C7AE0A73D34103BBAFC28DF
F5EFF54B907D95B1DF08E675550F13A4B6604FFC17CA288FD1CC7096D726C6B2
C054C7751CFC7AC8BF1ED6A32D66511AEED2FA6C37F117589FACCB71230099ED
F18616A45F1BFC054DCBF2EC0EA1161B068E764A4E1DE82AEA5A3E8E8BC2FF43
2AB25FA09826009C4E4715E197E2EE9ED128ED2BB342DE97667709F7B28D45C6
38B1E5186AC11D136CC3448E9AA6D4C4770E08E841776F6509E0ED7BE47AE414
63D9C6E8E7F01358B19E81946781DC46AD622636645323AC9E8F54DE259C72AD
22897189FE0F94DACB63A12DCC91B36ABC6DF0B40893506F3140928B9655EB7D
EF30A3DB087744731F1B354F905087C09442F619A708632B420698677F88B48A
8E2F10E9A1BFD691701F3F9A7F4CA7AE962B35932B420A169FFC7D68D73BA28A
2E4EDA6F7AF4043AC8A0504DA03A807E596A33D931C86DBD515DF7785F37DDD4
474042CC4F4162256A72D27DCD5652CA25875780DC8D08C77D206AC4B2283BDC
ACE78B97B16C36FAAC198AC144BBB28667AD4735F92BF68422AB02503228222A
FE28DF3EA257CAC35C50DA04BDFF02BCC2A768ED0E596CD52A5BE194D1DEDDC5
E2C98671187FC42A5266E0FE0F5E54CED9DC3033002EA38F173D29C447057E67
995899943096075A9679EB9DD294E7F21BF963B1740125A70AA528B388FB3667
F42E599A3A9E8E61BAD3C619761018607BAE01DE1CFF320F3157A5DEB0B498A8
D671A1AA6B00EBBF4D02E0F45AECE3057192DEA53037DDA7038B291BD520884D
9BC7EA66A90104E043283964DC49C82D33DF5556D6D146EC25098ACCEF8F0C77
5A69959A716846A607767E0EDAFE626D21BC6D788089AE7E76E66FA49DC0141F
A2284C69EAA48BCFE3558619736613D444EDF76F2B6CCA06BB6235DBFA51338D
A657E8F409A216DF9BB9B49A59D093849383726A4847D252EE73860F3BB1FCE7
56184C9D19E3AC4A3D79926098032603A03A371E332BF4B19AC4D0A15A41274A
AFF19A9931FAA3374D58C00F1CADF8954F84C8BDA1DBBDE3E70B4D4BDA59A980
C1BC8922D7712DA33D6055BC37F23BD442607D15C734AECBDE7A320DD31ECDBA
C3D9438059E073834DF1AB975081D63E12340D6D8893640300AC99FA9028EAAE
5F5BEB1A885DA0C978AAA9CA31072EF21EA6110D608D7C7AD0DC1ABAA5A2CA89
7BD4CE651950AB1482F1CE5704D27925AD
0000000000000000000000000000000000000000000000000000000000000000
0000000000000000000000000000000000000000000000000000000000000000
0000000000000000000000000000000000000000000000000000000000000000
0000000000000000000000000000000000000000000000000000000000000000
0000000000000000000000000000000000000000000000000000000000000000
0000000000000000000000000000000000000000000000000000000000000000
0000000000000000000000000000000000000000000000000000000000000000
0000000000000000000000000000000000000000000000000000000000000000
cleartomark
%%EndResource
%%EOF
