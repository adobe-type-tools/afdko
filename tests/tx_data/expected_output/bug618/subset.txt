StartFontMetrics 2.0
Comment Copyright 2018 Adobe Systems Incorporated. All Rights Reserved.
Comment Creation Date: Fri Oct 26 21:28:00 2018
FontName SourceSerifPro-Regular
FamilyName Source Serif Pro
ItalicAngle 0
IsFixedPitch false
FontBBox 45 -13 560 669
UnderlinePosition -100
UnderlineThickness 50
Version 1.0
Notice Source is a trademark of Adobe Systems Incorporated in the United States and/or other countries.
StartCharMetrics 2
C 97 ; WX 509 ; N a ; B 45 -13 503 488 ;
EndCharMetrics
EndFontMetrics
